### PR TITLE
DS-273 algorithm handler bug fix for fifo pipe

### PR DESCRIPTION
**Pull request by zeryx**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: algorithmia
 Type: Package
 Title: Allows you to Easily Interact with the Algorithmia Platform
-Version: 0.1.0
+Version: 0.2.0
```
Comment (jamesatha) [inline DESCRIPTION:4]: This was never published, right? so we can keep the same version?

File: R/AlgorithmHandler.r [hunk 2]
```diff
@@ -56,9 +56,10 @@ AlgorithmHandler <- methods::setRefClass(
         }
       }
       # Begin startup
-      outputFile <- fifo("/tmp/algoout", open="w", blocking=TRUE)
-      inputFile <- file(pipeName)
+      outputFile <- fifo("/tmp/algoout", blocking=TRUE)
+      inputFile <- file("stdin")
```
Comment (jamesatha) [inline R/AlgorithmHandler.r]: isn't this supposed to be pipeName for testing purposes?

Comment (zeryx) [reply R/AlgorithmHandler.r]: yes, good catch - thanks updating.

File: DESCRIPTION (hunk 1 above)
Comment (zeryx) [reply DESCRIPTION:4]: 0.1.0 was published, 0.2.0 was - I pushed that to master accidentally so it should just null in the merge

**Review by jamesatha — APPROVED** [review]
sure

**Pull request merged**